docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Messaging_JDBC_Store_for_Messaging_Journal.adoc

```diff
@@ -50,12 +34,28 @@ The name of these tables can be configured with the
 `journal-bindings-table`, `journal-messages-table`,
 `journal-page-store-table, and journal-large-messages-table`.
 
+Please note that the configuration of the underlying pool is something that you need to take care of.
+You  need at least four connections:
+
+ * one for the binding
+ * one for the messages journal
+ * one for the lease lock (if you use HA)
+ * one for the node manager shared state (if you use HA)
+
+So you should define a `min-pool-size` of 4 for the pool. +
```

Comment:
It's not clear where this attribute could be found from the documentation. `/subsystem=datasources/data-source=NAME`?